setup.py

```diff
@@ -89,7 +89,7 @@
 # In brief, numpy 1.14.0 introduced whitespace changes that impact repr
 # on doctest
 install_requires = ["click", "numpy >= 1.9.2, < 1.14.0", "future >= 0.16.0",
-                    "scipy >= 0.13.0", 'pandas >= 0.19.2',
+                    "scipy >= 0.13.0", 'pandas >= 0.20.0',
```

Comment:
Just noticed that .travis.yml is not going to honor this version restriction since conda is installing pandas and biom is installed with the `--no-deps` flag. I think the Travis file needs to change.